README.md

```
@@ -132,6 +134,37 @@ r2c --no-gitignore . --output full-context.md
 
 # Process multiple files (up to 5 files/directories)
 r2c file1.go file2.go file3.go
+
+# Use configuration file for defaults (CLI flags override)
+r2c .
+```
+
+### Configuration File
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)